scripts/exobsmon_plot.sh

```diff
@@ -20,87 +20,58 @@ if [[ ! -e ${chan_yaml} ]]; then
    exit 2
 fi
 
-#-----------------------------------------------------------
-# split $plot_yaml into sat/instr[/plot], minimization, obs
+#---------------------------------------------------------------
+# split $plot_yaml into sat/instr[/plot], minimization, and obs
+# in order to reduce the plot jobs to a more managable size 
 #
 ${APRUN_PY} ${USHobsmon}/splitPlotYaml.py -i ${plot_yaml} -c ${chan_yaml}
 
-
 #--------------------------------------------------------------
-# Submit OM_sat_plots job if split yields any sat_*.yaml files
+# Submit OM_plots job if split yields any *.yaml files
 #
-if compgen -G "${DATA}/sat_*.yaml" > /dev/null; then
+if compgen -G "${DATA}/*.yaml" > /dev/null; then
 
-   jobname="OM_sat_plots"
-   logfile="${OM_LOGS}/${MODEL}/OM_sat_plot.log"
+   jobname="OM_plots"
+   export logfile="${OM_LOGS}/${MODEL}/OM_plot.log"
    if [[ -e ${logfile} ]]; then rm ${logfile}; fi
 
-   cmdfile="OM_sat_jobscript"
+   cmdfile="OM_jobscript"
    >$cmdfile
-   ctr=0
-
-   for yaml in ${DATA}/sat_*.yaml; do
-      echo "${ctr} $yaml"
-      echo "${ctr} ${APRUN_PY} ${USHobsmon}/plotObsMon.py -i ${yaml}  -p ${PDATE}" >> $cmdfile
-      ((ctr+=1))
-   done 
-   chmod 755 $cmdfile
 
-   echo "ctr: $ctr"
-   echo "submitting job ${jobname}"
-
-   if [[ ${ctr} > 0 ]]; then
-      $SUB --account ${ACCOUNT} -n ${ctr}  -o ${logfile} -D . -J ${jobname} --time=1:00:00 \
-           --mem=80000M --wrap "srun -l --multi-prog ${cmdfile}"
-   fi
-
-fi
-
-#------------------------------------------------------------------
-# Submit OM_min_plots job if split yields a minimization.yaml file
-#
-if compgen -G "${DATA}/minimization.yaml" > /dev/null; then
-
-   jobname="OM_min_plots"
-   logfile="${OM_LOGS}/${MODEL}/OM_min_plot.log"
-   if [[ -e ${logfile} ]]; then rm ${logfile}; fi
-
-   cmdfile="OM_min_jobscript"
-   echo "0 ${APRUN_PY} ${USHobsmon}/plotObsMon.py -i ${DATA}/minimization.yaml  -p ${PDATE}" > $cmdfile
-   chmod 755 $cmdfile
-
-   echo "submitting job ${jobname}"
-
-   $SUB --account ${ACCOUNT} -n 1  -o ${logfile} -D . -J ${jobname} --time=0:05:00 \
-        --mem=80000M --wrap "srun -l --multi-prog ${cmdfile}"
-fi
-
-#------------------------------------------------------------------
-# Submit OM_con_plots job if split yields any obs_*.yaml files
-#
-if compgen -G "${DATA}/obs*.yaml" > /dev/null; then
-   echo "have OBS plots"
-   jobname="OM_obs_plots"
-   logfile="${OM_LOGS}/${MODEL}/OM_obs_plot.log"
-   if [[ -e ${logfile} ]]; then rm ${logfile}; fi
-
-   cmdfile="OM_obs_jobscript"
-   >$cmdfile
    ctr=0
-
-   for yaml in ${DATA}/obs*.yaml; do
-      echo "${ctr} $yaml"
-      echo "${ctr} ${APRUN_PY} ${USHobsmon}/plotObsMon.py -i ${yaml}  -p ${PDATE}" >> $cmdfile
+   for yaml in ${DATA}/*.yaml; do
```

Comment:
I worry this might not be "safe" enough and perhaps we want a prefix on the YAML or something to ensure it isn't trying to run things it shouldn't?

Reply:
Fair point.  I'll see what I can do.

Reply:
The resulting yaml files can include:

- sat_[satname]_[instrument].yaml
- sat_[satname]_[instrument]_[ctr].yaml (sat/instruments with lots of channels)
- minimization.yaml
- observations.yaml

What if I precede the file names with `OM_` and then do
`for yaml in ${DATA}/OM_*yaml; do`
Would that be safe enough?

Reply:
yeah I think that's good enough, and we can cross a future bridge if we need to.

Reply:
`splitPlotYaml.py` now uses a prefix of `OM_PLOT_` to uniquely identify the resulting yaml files, and `exobsmon_plot.sh` has been modified to search for `OM_PLOT*yaml`.